facebook/delphiFacebook/R/responses.R

```diff
@@ -535,8 +539,8 @@ create_complete_responses <- function(input_data, county_crosswalk)
     "G1", "G2", "G3", "H1", "H2", "H3", "I1", "I2", "I3", "I4", "I5",
     "I6_1", "I6_2", "I6_3", "I6_4", "I6_5", "I6_6", "I6_7", "I6_8",
     "I7", "K1", "K2", "V11a", "V12a", "V15a", "V15b", "V16", "V3a", "module", # added in Wave 11
-
-    "token", "wave", "UserLanguage",
+    
+    "raceethnicity", "token", "wave", "UserLanguage",
```

Comment:
This will cause the count of `!is.na()` responses in `filter_complete_responses` to be wrong, since we're adding a new column that's not from Qualtrics; to ensure we continue to filter the same way, we'll need to fix that

(and we should add a test with raceethnicity data that ensures responses with ≥2 answers are kept and those with < 2 are dropped)